opm/autodiff/BlackoilModelEbos.hpp

```diff
@@ -183,6 +183,7 @@ namespace Opm {
             } else {
                 // set the initial solution.
                 ebosSimulator_.model().solution( 1 /* timeIdx */ ) = ebosSimulator_.model().solution( 0 /* timeIdx */ );
+                ebosSimulator_.model().shiftIntensiveQuantityCache(/*numSlots=*/1);
```

Comment:
good catch!

possibly it is better to call 

```
ebosSimulator_.model().advanceTimeLevel();
```

because this does not require to do the shift of the solution manually. anyway, this is good to go as far as I'm concerned.

Reply:
I considered that as well, but I went for this approach to make it more visible in the code. But you are probably right. These are ebos details. 
Then I think we should also call updateFailed() above. I will update the PR.

Reply:
> Then I think we should also call updateFailed() above.

you're right, I missed that. You slowly know your ways around in my code better than myself ;)